packages/core/scripts/collateral-umip/1_Propose.js

```diff
@@ -97,7 +107,12 @@ async function runExport() {
   console.log(`Sending to governor @ ${governor.address}`);
 
   // Send the proposal
-  const txn = await governor.propose(transactionList, { from: proposerWallet, gas: 2000000 });
+  await gasEstimator.update();
+  const txn = await governor.propose(transactionList, {
+    from: proposerWallet,
+    gas: 2000000,
```

Comment:
Do you need the `gas` param? I think the `gas` should be auto filled

Reply:
https://github.com/UMAprotocol/protocol/pull/2727#discussion_r596515482